hazelcast/src/test/java/com/hazelcast/map/BasicMapTest.java

```diff
@@ -1191,6 +1194,26 @@ public void testIfWeCarryRecordVersionInfoToReplicas() {
     }
 
     @Test
+    public void github_11489() throws Exception {
+        HazelcastInstance hz = createHazelcastInstance();
+        IMap test = hz.getMap("test");
```

Comment:
Should be `IMap<Integer, Integer> test = hz.getMap("test");` (no need for a generics warning here).